### PR TITLE
Newyork branch

### PR DESCRIPTION
**Pull request by haraphat01**
Project 3: Clone NewYorkTimes page.
In this project we:

Layout the structure of the page with semantic HTML elements!
Fill in those elements with the content.
Style the page to look like the new york times page.
Fix after code review
CSS Linter errors

Remove inline comments
Edited the PR
Remove inline styles from css
configure stickler

### CODE REVIEW

**Comment by mikenath223** [comment]
### Pull request currently under review by Michgolden Ukeje (TSE).
Kindly exercise patience 👋

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hi Team 👋

Good job on this. @haraphat01  and @uimarshall 
This is quite an inch-perfect representation of the original webpage.👍

However, a few changes need to be made before proceeding: 

- [ ] Required - Kindly add a pull request message, and make it descriptive enough. It should describe the implementations you made in the project.

- [ ] Required - Also kindly ensure you remove the inline comments in your code. If you want to refer to a previous version of your code consider using git history.

- [ ] Required - In addition, remove the inline style that exists in your HTML file.

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

HAPPY CODING👨‍💻

File: index.html [hunk 1]
```diff
@@ -0,0 +1,791 @@
...
+          Clement Pryke of the University of Minnesota, Jamie Bock of the
+          California Institute of Technology and Dr. Kuo of Stanford.</p>
+         
+          <p>“The South Pole is the closest you can get to space and still be on
+          the ground,” Dr. Kovac said. He has been there 23 times, he said,
+          wintering over in 1994. “I’ve been hooked ever since,” he said.</p>
+       
+        
+        <figure>
+          <img src="images/sat.jpg" alt="dish" style="width: 100%;" />
```
Comment (mikenath223) [inline index.html]: Kindly remove the inline style that exists here.

**Comment by haraphat01** [comment]
**Clone NewYorkTimes page.**
In this project we:

- Layout the structure of the page with semantic HTML elements!
- Fill in those elements with the content.
- Style the page to look like the new york times page.

**Fix after code review**
CSS Linter errors

- Remove inline comments
- Edited the PR 
- Remove inline styles from css
- configure stickler

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes ♻️ 🚥 

Hi **TEAM**, you have done a great job working on the changes requested by the previous TSE 👍 However, we need to make a few changes before proceeding. Kindly check comments below:
- The previous TSE made mention of editing the PR. Kindly edit it.
- Kindly enable stickler for the repo by visiting stickler-ci.com
- There are some repeated codes in our CSS file, let us apply the DRY rule


Kindly make these changes and submit another code review 😄 
Happy coding_ 👨‍💻 
Daniel Larbi Addo on slack

**Comment by addod19** [comment]
## Pull request under review by Daniel Larbi Addo (TSE)

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes ♻️ 🚥 

Hi **TEAM**, we have to address the comments by the TSE's before we can progress. Kindly take time and go through the comments, and if you have any problem, kindly contact me on slack Daniel Larbi Addo

Kindly address the previous comments and submit another code review

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hello again Team  👋

Your project looks inch-perfect. 👌💪


However as stated in the previous review, kindly update your pull request by editing it. 
Simply do this click the three dots icon at the top of the pull request which is circled in red in the screenshot below then click edit, and edit the pull request with a message that describes the implementations you made in this project.[Screenshot]👇
Inkedice_screenshot_20200414-130953_LI


Awesome project once more.👏👏

You are almost there.👍

Kindly fix these issue and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

HAPPY CODING👨‍💻

**Comment by uimarshall** [comment]
> ### Hello again Team 👋
> Your project looks inch-perfect. 👌💪
> 
> However as stated in the previous review, kindly update your pull request by editing it.
> Simply do this click the three dots icon at the top of the pull request which is circled in red in the screenshot below then click edit, and edit the pull request with a message that describes the implementations you made in this project.[Screenshot]👇
> Inkedice_screenshot_20200414-130953_LI
> 
> Awesome project once more.👏👏
> 
> You are almost there.👍
> 
> Kindly fix these issue and request another review👍
> Feel free to contact me on slack(michgolden) if you need further clarification.
> 
> HAPPY CODING👨‍💻

Changes affected, Thanks for the awesome review.

**Review by mikenath223 — APPROVED** [review]
### CONGRATULATIONS  TEAM 👏👏👏

This project meets the requirements for approval.

**STATUS: APPROVED ✅**


Don't forget to fill the project completion form👍
Good luck with the next project.

HAPPY CODING👨‍💻

**Pull request merged**